packages/core/src/checkbox/Checkbox.tsx

```diff
@@ -213,7 +213,7 @@ export const Checkbox = forwardRef<HTMLLabelElement, CheckboxProps>(
           validationStatus={validationStatus}
           error={error}
         />
-        {label}
+        <span className={withBaseName("label")}>{label}</span>
```

Comment:
this might not be necessary, don't think this class was used

Reply:
not sure i understand - this is for the styling i've added for the label ?